src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -149,7 +149,7 @@ public async Task FishUntilMessageAsync<T>(TimeSpan? max = null, CancellationTok
                     return true; // please continue receiving, don't stop
                 },
                 cancellationToken: cancellationToken)
-                .ConfigureAwait(false).WithCancellation(cancellationToken);
+                .WithCancellation(cancellationToken);
```

Comment:
We don't need `.WithCancellation()` here, token is already passed into the method.